connectors/sources/sharepoint_online.py

```diff
@@ -748,6 +798,80 @@ def _prefix_email(email):
     return _prefix_identity("email", email)
 
 
+def _postfix_group(group):
+    if group is None:
+        return None
+
+    return f"{group} Members"
+
+
+def is_domain_group(user_fields):
```

Comment:
Do they need to be in the module?

I see these are functions that are pure, which is nice, but idk if they should be here. I'm also not a Python expert and not sure what are best practices here.

Reply:
I think they should as they're specific to SPO. That's also a suggestion by the IDE to create a top-level function, if it's a pure function.

Reply:
Personal preference, if it's only one line, I don't think it makes much sense to make it a function.